flags.go

```diff
@@ -341,6 +345,69 @@ type flagStorage struct {
 	DebugMutex      bool
 }
 
+const GCSFUSE_PARENT_PROCESS_DIR = "gcsfuse-parent-process-dir"
+
+// 1. Returns the same filepath in case of absolute path or empty filename.
+// 2. For relative path with . or .. or nothing, it resolves smartly the path
+// in case of child proces it resolves with respect to GCSFUSE_PARENT_PROCESS_DIR
```

Comment:
please add the details why the child process resolves with respect to GCSFuse_parent_process_dir